src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java

```diff
@@ -152,7 +153,15 @@ public GetTranscriptionByIdResponse mapToTranscriptionResponse(TranscriptionEnti
         }
         if (nonNull(transcriptionEntity.getTranscriptionUrgency())) {
             transcriptionResponse.setUrgency(transcriptionEntity.getTranscriptionUrgency().getDescription());
+
+            // populate the urgency details
+            TranscriptionUrgencyDetails urgencyDetails = new TranscriptionUrgencyDetails();
+            urgencyDetails.setPriorityOrder(transcriptionEntity.getTranscriptionUrgency().getPriorityOrder());
+            urgencyDetails.setTranscriptionUrgencyId(transcriptionEntity.getTranscriptionUrgency().getId());
```

Comment:
The setUrgency line above (line 155) that this new field is superseding has a null check on transcription urgency, do we need a similar check here?